platforms/aws-graviton/run.sh

```diff
@@ -12,8 +12,6 @@ ssh -i ./server-key.pem ec2-user@ec2-54-209-135-27.compute-1.amazonaws.com -tt "
     export CI=\"true\"
     export PRISMA_CLIENT_ENGINE_TYPE=\"$PRISMA_CLIENT_ENGINE_TYPE\"
 
-    rm -rf ./node_modules;
```

Comment:
This is due to `rsync --exclude "node_modules"`, right?

Reply:
exactly, i wanted to preserve local `node_modules` so we could run `pnpm store prune` in finally. `pnpm` gets installed in `node_modules` and if we keep this line, it will be gone by the time we do a cleanup.